### DISCUSSION TITLE
scanning for cells

### DISCUSSION THREAD

**Discussion by brcisna** [danielewood/sierra-wireless-modems#89] [Q&A]
Hello All,

After a successfull flash of the em7455, didn't have much time to mess with the modem after this but did try doing a "scan" in the modem  gui,,using Debian Bullseye. I did not have the actual antennas with me to screw onto the sma studs. Our building is only about half mile from a cell tower. I could not see any cells avaliable. I am probably missing something on this. I should not even need a sim in to do this should I? Going to do some more thrashing tomorrow,,but wanted someone with some experience  in this to get guided a little.  I did find the AT command to do this but didnt have time to try the AT command.
I guess what am wondering is ,if a scan never finds any cells available i am never going to be able to get service.

Thanks

**Comment by danielewood** [comment]
Without any antennas connected, you are unlikely to be able to see any signals, much less perform the handshake needed to register your modem with the network. 

If you detach the antenna leads and try to connect to WiFi, you might be able to see a network from a few inches away, but certainly not through a wall.
